app/src/main/java/com/codesoom/assignment/application/ProductNotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package com.codesoom.assignment.application;
+
+
+import org.springframework.http.HttpStatus;
+
+/** 상품을 찾지 못한 경우 해당 예외를 던집니다. */
```

Comment:
`해당`, `예외`는 없어도 될 것 같습니다.
```suggestion
/** 상품을 찾지 못하면 던집니다. */
```